index.html

```diff
@@ -79,28 +79,30 @@ <h1>
                 <div class="row-title">
                     <h4><span>About</span> Me</h4>
                 </div>
-                    <h5>i am a ui/ux designer and a human</h2>
-                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni ea blanditiis sunt fugiat nam
-                            officia
-                            ut rerum culpa quam odio ipsa itaque, impedit laboriosam non eaque numquam magnam. Veniam,
-                            aperiam.
-                        </p>
-                        <div class="info">
-                            <div class="info left col-3">
-                                <ul>
-                                    <li>Name: <span>John Doe</span></li>
-                                    <li>Age: <span>27</span></li>
-                                </ul>
-                            </div>
-                            <div class="info right col-3">
-                                <ul>
-                                    <li>Email: <span>John_Doe@gmail.com</span></li>
-                                    <li>Location : <span>Los Angeles, USA</span></li>
-                                </ul>
-                            </div>
-                        </div>
-                        <input type="submit" value="Dowload resume" class="btn">
-                        <input type="submit" value="Hire me" class="btn">
+                <h5>i am a ui/ux designer and a human</h2>
```

Comment:
skirias priekis nuo galo:)
<img width="388" alt="Capture1" src="https://user-images.githubusercontent.com/73877213/101990863-9ca6ff00-3cb1-11eb-8919-05c134855997.PNG">

Reply:
Like shaggy would say "it wasn't me" :D net nepastebejau, bet istaisau 👍